pom.xml

```diff
@@ -94,7 +94,14 @@
 			<plugin>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
+					<debug>true</debug>
+					<optimize>true</optimize>
 					<annotationProcessorPaths>
+						<path>
```

Comment:
Ah, I guess we do need to keep the `annotationProcessPaths` section, and thus the `maven-compiler-plugin` block as a whole. But good that we can have it in the toplevel POM.

Reply:
Yup!